build_defs/java_opts.bzl

```diff
@@ -69,7 +202,7 @@ def protobuf_versioned_java_library(
     """
     osgi_java_library(
         javacopts = JAVA_OPTS,
-        automatic_module_name = automatic_module_name,
+        module_name = module_name,
```

Comment:
All instances of `Automatic-Module-Name` are replaced with a `module-info` (matching the same module name) and the manifest entry `Multi-Release: true`